compose.yml

```diff
@@ -1,12 +1,24 @@
 ---
 # This playbook composes OSTree
+
+- include_vars:
+    file: vars/packages.yml
+
 - name: Clone Fedora-Atomic buildscripts
   git:
     repo=https://pagure.io/fedora-atomic.git
     version=f25
     depth=1
     dest=/workspace/fedora-atomic
 
+- name: Customize Packages
+  lineinfile:
```

Comment:
this works, but maybe writing to a new file... and composing this file would be a cleaner approach. WDYT?